metaflow/runtime.py

```diff
@@ -160,13 +163,24 @@ def __init__(
             for deco in step.decorators:
                 deco.runtime_init(flow, graph, package, self._run_id)
 
+    def _update_rerun_steps(self):
+        """
+        Any steps following steps to be rerun should also be included as
+        "rerun" steps. This is to ensure that the flow is executed correctly.
+        """
+        for step_name in self._graph.sorted_nodes:
+            if step_name in self._rerun_steps:
```

Comment:
we probably need to run this until stability right?

Reply:
not sure what you mean "stability" here, the sorted nodes are in topological order already.